app/config/tables/gridScreen/forms/gridScreen/comparison_screen.handlebars

```diff
@@ -23,10 +23,18 @@
     
     
 <style>
+     @media (min-width:478px) {
+        .odk-container{
+            width: 95vw;
+            margin: 0 auto 0 auto ;
+        }
+    }
     table {
         padding: 15px; 
         border-collapse:collapse; 
-        width:100%;
+        max-width: 400px;
```

Comment:
I don't think we want a max-width because there are multiple size screens on Android. Here is an example  10 inches. https://www.lenovo.com/us/en/tablets/android-tablets/lenovo-tab-series/Tab-M10-HD-2nd-Gen/p/ZA6W0178US

You can use the Android emulator to create bigger screens and see how your changes look.
@linl33  thoughts?

Reply:
There are two ways we could fixed the table overflow issue. I first opted for table-layout: fixed but then we had to go for word-wrap: break-word for the table header which didn't look much better.

Reply:
Besides, I launched the live server for this checked. If we don't want then we can change the max-width compatible to a big screen.

Reply:
I agree it's probably best to avoid setting a `max-width`. But this is only in the sample, I don't think it matters that much.